php-checks/src/main/java/org/sonar/php/checks/utils/CheckUtils.java

```diff
@@ -66,6 +66,15 @@ public static boolean isStaticClassMember(List<AstNode> modifiers) {
     return false;
   }
 
+  public static boolean hasModifier(List<SyntaxToken> modifiers, String toFind) {
```

Comment:
@vilchik-elena I suppose that this method could be used by other checks or other methods e.g.  FunctionUtils.hasModifier(...)